assets/js/modules/reader-revenue-manager/components/common/PublicationSelect.js

```diff
@@ -130,7 +130,10 @@ export default function PublicationSelect( props ) {
 				( { publicationId, displayName } ) => (
 					// eslint-disable-next-line sitekit/acronym-case
 					<Option key={ publicationId } value={ publicationId }>
-						{ displayName }
+						{
+							// eslint-disable-next-line sitekit/acronym-case
+							`${ displayName } (${ publicationId })`
+						}
```

Comment:
This string should be translated:

```suggestion
						{ sprintf(
							/* translators: 1: Publication display name, 2: Publication ID */
							__( '%1$s (%2$s)', 'google-site-kit' ),
							displayName,
							// eslint-disable-next-line sitekit/acronym-case
							publicationId
						) }
```